packages/iTwinUI-react/src/core/Table/filters/FilterToggle.test.tsx

```diff
@@ -0,0 +1,68 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+
+import { render, screen } from '@testing-library/react';
+import React from 'react';
+import { FilterToggle } from './FilterToggle';
+import { tableFilters } from './tableFilters';
+
+it('should display active filter', () => {
+  const column = {
+    canFilter: true,
+    filterValue: 'value',
+    render: jest.fn(),
+    setFilter: jest.fn(),
+    Filter: tableFilters.TextFilter(),
+  } as any;
+
+  render(<FilterToggle column={column} />);
```

Comment:
Can you create `renderComponent` function that has default props? You can check our other test files.

Reply:
yep